### PR TITLE
YALB 634 wire footer

### PR DESCRIPTION
**Pull request by codeb-a**
### Description of work
- Adds social links block to footer region

### Functional testing steps:
- [ ] login 
- [ ] Navigate to footer

### CODE REVIEW

**Review by ModulesUnraveled — APPROVED** [review]
Looks good. :shipit:

File: templates/ys/ys-social-links.html.twig [hunk 1]
```diff
@@ -0,0 +1 @@
+{% include '@molecules/social-links/social-links.twig' %}
```
Comment (ModulesUnraveled) [inline templates/ys/ys-social-links.html.twig:1]: I just realized that we didn't consider what happens when no social media fields are filled out. I expect it'll still print empty markup to the page. We should check that, and add an `if` statement or something to only include this template if at least one of the fields is filled in.

**Review by ModulesUnraveled — CHANGES_REQUESTED** [review]

**Review by ModulesUnraveled — APPROVED** [review]
That does it. Thanks!

**Comment by nJim** [comment]
Merging this work into develop as I'm preparing for demo. Great job.

**Pull request merged**